Orleans.Providers.MongoDB/MongoDBSiloExtensions.cs

```diff
@@ -100,6 +100,7 @@ public static ISiloBuilder UseMongoDBClient(this ISiloBuilder builder, Func<ISer
         public static IServiceCollection AddMongoDBReminders(this IServiceCollection services,
             IConfiguration configuration)
         {
+            services.AddReminders();
```

Comment:
Can you please add it to the other overload above and finally remove `AddReminders` call from the integration test.